.tx/config

```diff
@@ -1,6 +1,6 @@
 [main]
 host = https://www.transifex.com
-lang_map = zh_CN:zh-cn, zh_TW:zh-tw, pt_BR:pt-br
+lang_map = zh_CN:zh-cn, zh_TW:zh-tw, pt_BR:pt-br, es_419:es-419, aa_DJ:aa-dj
```

Comment:
My one thought: any chance you could add a comment to this line to help clarify what it means? What are the variations that only differ by capitalization?